extensions/vertx-http/deployment/pom.xml

```diff
@@ -86,6 +102,89 @@
                     </annotationProcessorPaths>
                 </configuration>
             </plugin>
+            
+            <!-- So we don't have to keep a version of these javascript libs in our repo -->
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <version>3.1.2</version>
```

Comment:
The version of the plugin should be defined in the `build-parent` if not defined already.